src/org/scada_lts/dao/model/multichangehistory/MultiChangeHistoryValues.java

```diff
@@ -0,0 +1,140 @@
+package org.scada_lts.dao.model.multichangehistory;
+
+import java.util.Objects;
+
+/**
+ * @author  grzegorz.bylica@abilit.eu on 16.10.2019
+ */
+public class MultiChangeHistoryValues {
+
+    private int id;
+    private int userId;
+    private String userName;
+    private String viewAndCmpIdentyfication;
+    private String interpretedState;
+    private long ts;
```

Comment:
Is it possible to change variable name ts to timeStamp?

Reply:
improved